Matveibos/hometask-3/rainfall_stat.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# this class is used for calculate mean and variance values for town rainfall
+class RainfallStat
+  SEPARATOR = ':'
+  REGULAR = /\d*\.\d/.freeze
+
+  def mean
+    create_hash
```

Comment:
You need to have `@processed_data` variable populated for `#mean` and `#variance` methods. Why not to call this method in the initializer, rather than calling it two times in methods?